### ISSUE TITLE
Support for windows 10?

### ISSUE THREAD

**Issue by kevinrademan** [natevoci/KeyboardRedirector#2]
When starting up the application in windows 10 it appears to break while reading values from the registry. 
```
Additional information: Object reference not set to an instance of an object. item=HID#VID_1B1C&PID_0C04#7&bc0c31&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}, OurKey found, retryCount=0, deviceDesc=@input.inf,%hid_device_vendor_defined_range%;HID-compliant vendor-defined device, deviceClass=
```

Does this application support windows 10?

**Comment by natevoci** [comment]
Hi. Sorry for the delay.

Looks like MS removed one of the registry values it was relying on in windows 10, so I've just pushed commit 2363572ef2c3c5e30a7f6aafa42a15262617890c to get it working in windows 10. Works on my machine anyway.

If you're not building it yourself then I've uploaded the new version to my website at http://nate.deepcreek.org.au/dev/keyboardredirector.rails but my VS 2010 environment is broken so I had to upgrade to VS 2015. Hopefully it doesn't make any difference, but it's pretty much untested so if you find any problems let me know.
